src/ngx_http_lua_rewriteby.c

```diff
@@ -125,6 +125,13 @@ ngx_http_lua_rewrite_handler(ngx_http_request_t *r)
                 }
 
                 return NGX_HTTP_OK;
+
+            } else {
```

Comment:
Seems like the previous branch always returns and never falls through, and we can save the `else` here.